haystack/components/generators/chat/hugging_face_api.py

```diff
@@ -2,6 +2,8 @@
 #
 # SPDX-License-Identifier: Apache-2.0
 
+import json
+import logging
```

Comment:
let's use Haystack logger: `from haystack import logging`

unit tests are failing for this reason